ansible/vagrant-playbook.yml

```diff
@@ -1,4 +1,11 @@
 ---
+- hosts: all
+  gather_facts: no
```

Comment:
Same thing here as the other playbook, with respect to comments / provisioning with Python 3.